tests/chainer_tests/functions_tests/math_tests/test_minmax.py

```diff
@@ -190,163 +108,80 @@ def test_duplicate_axis(self):
             functions.Max((0, 0))
 
     def test_pos_neg_duplicate_axis(self):
+        x_data = numpy.random.uniform(-1, 1, (3, 2, 4)).astype(numpy.float32)
+        x = chainer.Variable(x_data)
         with self.assertRaises(ValueError):
-            self.x.max(axis=(1, -2))
```

Comment:
Good catch...

Reply:
Thanks.  `ValueError` has been raised by `numpy.ndarray.max`, not by `chainer.functions.max`.